frontend/src/modules/automation/pages/automation-list-page.vue

```diff
@@ -1,154 +1,187 @@
+            {{ option.label }}
+          </div>
         </div>
+        <p class="pl-4 text-xs leading-5 text-gray-500">
+          {{ pluralize('automation', totalAutomations, true) }}
+        </p>
+      </div>
+      <div>
+        <el-popover
+          trigger="click"
+          placement="bottom-end"
+          width="15rem"
+          popper-class="!p-2"
+        >
+          <template #reference>
+            <el-button class="btn btn--primary btn--sm flex items-center !py-2">
+              Add automation
+              <span class="ri-arrow-down-s-line text-base ml-2 flex items-center h-4" />
+            </el-button>
+          </template>
+          <div class="popover-item h-auto mb-1 py-2 px-2.5" @click="createAutomation('webhook')">
+            <div class="flex">
+              <div class="mt-0.5">
+                <img alt="Webhook" src="/images/automation/webhook.png" class="w-4 max-w-4">
+              </div>
+              <div class="pl-2">
+                <h6 class="text-xs leading-5 font-medium mb-0.5 text-gray-900">
+                  Webhook
+                </h6>
+                <p class="text-2xs leading-4.5 text-gray-500 break-words">
+                  Send webhook payloads to automate workflows
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="popover-item h-auto py-2 px-2.5" @click="authenticateSlack">
+            <div class="flex">
+              <div class="mt-0.5">
+                <img alt="Slack" src="https://cdn-icons-png.flaticon.com/512/3800/3800024.png" class="w-4 max-w-4">
+              </div>
+              <div class="pl-2">
+                <h6 class="text-xs leading-5 font-medium mb-0.5 text-gray-900">
+                  Slack notification
+                </h6>
+                <p class="text-2xs leading-4.5 text-gray-500 break-words">
+                  Send notifications to your Slack workspace
+                </p>
+                <el-button
+                  v-if="!slackConnected"
+                  class="btn btn--primary btn--sm !h-8 mt-3"
+                >
+                  Install app
+                </el-button>
+              </div>
+            </div>
+          </div>
+        </el-popover>
       </div>
-
-      <!-- Webhooks list -->
-      <app-automation-list-table
-        class="pt-4"
-        @open-executions-drawer="onOpenExecutionsDrawer"
-        @open-edit-automation-drawer="
-          onOpenEditAutomationDrawer
-        "
-      />
     </div>
+    <div
+      v-if="loadingAutomations"
+      v-loading="loadingAutomations"
+      class="app-page-spinner"
+    />
+    <app-automation-list-table
+      v-else-if="totalAutomations > 0"
+      class="pt-4"
+      @open-executions-drawer="showAutomationExecutions = $event"
+      @open-edit-automation-drawer="updateAutomation($event)"
+    />
 
-    <!-- Empty state for no webhooks configured -->
+    <!-- Empty state for no automations configured -->
```

Comment:
It's missing the empty state for slack notifications and webhooks. I believe this one should only be for when "All" tab is selected